### ISSUE TITLE
About the case where the loss is negative

### ISSUE THREAD

**Issue by Xiejiahao233** [Jiahao-UTS/DSLPT#3]
Hello，have you ever encountered a situation where the loss was negative during training?

**Comment by Jiahao-UTS** [comment]
The NLL loss function can be negative during training.

**Comment by Xiejiahao233** [comment]
Thanks a lot for your reply!
